src/js/components/Button/Button.js

```diff
@@ -253,6 +253,7 @@ const Button = forwardRef(
           {...rest}
           as={domTag}
           ref={ref}
+          active={active}
```

Comment:
can we just leave this as active prop ? without passing active ?

Reply:
otherwise some examples are still showing active without being moused over on calendar

Reply:
opps I put this change on line 96 when testing and didnt realize it was under  `<StyledButtonKind/>`
so you can ignore this comment